PSL/SENSORS/MPU6050.py

```diff
@@ -33,8 +33,8 @@ def __init__(self, I2C, **args):
         self.I2C = I2C
         self.ADDRESS = args.get('address', self.ADDRESS)
         self.name = 'Accel/gyro'
-        self.params = {'powerUp': ['Go'], 'setGyroRange': [250, 500, 1000, 2000], 'setAccelRange': [2, 4, 8, 16],
-                       'KalmanFilter': [.01, .1, 1, 10, 100, 1000, 10000, 'OFF']}
+        self.params = {'powerUp': None, 'setGyroRange': [250, 500, 1000, 2000], 'setAccelRange' : [2, 4, 8, 16],
+                       'KalmanFilter': 'double' }
```

Comment:
Good point. I haven't started modifying that part in the desktop app yet.
Here's my suggestion:
- if type(value) is None : 
      Assume that no arguments are required . Create a QMenuAction . 
- if type(value) == list :
     Pre-defined values have been provided. Create a submenu with clickable values
- if type(value == dict : 
    A user input is required, and this dictionary must have all relevant configuration options.
   The following keys can be created
   - 'dataType': 'double'/'integer'/'text' ...
   - 'min'/'max'/'value'    ... For sliders/spinboxes
   - Any other arguments required for various input types

Works?